### PR TITLE
1.3.3 beta1

### PR DESCRIPTION
**Pull request by diniska**
Library moved to cocoapods. Template project uses it.
.gitignore update to modern version
Images moved to assets. Missing launch image added to iOS 7.
NMRangeSlider and SDWebImage removed from repo.
Добавлена интеграция с travis: https://travis-ci.org/KosyanMedia/Aviasales-iOS-SDK.

### CODE REVIEW

**Comment by sevabill** [comment]
👍

File: Podfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Uncomment this line to define a global platform for your project
+ platform :ios, '7.0'
+# Uncomment this line if you're using Swift
+# use_frameworks!
+
+target 'AviasalesSDKTemplate' do
+
+pod 'SDWebImage', '~> 3.7'
```
Comment (glassoff) [inline Podfile]: а почему они не прописаны в зависимостях в podspec?

Comment (glassoff) [inline Podfile]: а, понял, это для темплейта

**Comment by glassoff** [comment]
👍

**Pull request merged**